saleor/graphql/order/mutations/draft_order_create.py

```diff
@@ -117,6 +117,9 @@ def clean_input(cls, info, instance, data):
             object_id=data.pop("shipping_method", None), raise_error=False
         )
 
+        if data.get("user_email", None):
+            data["user"] = None
```

Comment:
Does it still work if the email matches an existing user?

Reply:
The order resolvers, first try to fetch email from `order.user.email`, then as a fallback it tries to fetch `order.user_email`.

Reply:
My question is will it assign the correct user?

Reply:
@patrys right, it works on dashboard, because if provided email exists FE will send `user` key not ` user_email` but if someone will use playground and send existing email in `user_email` it will not assign that user to order. I will fix it.